libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -284,23 +292,6 @@ export class RushConfiguration {
    */
   public readonly commonFolder: string;
 
-  /**
-   * The folder where Rush's additional config files are stored.  This folder is always a
-   * subfolder called `config\rush` inside the common folder.  (The `common\config` folder
-   * is reserved for configuration files used by other tools.)  To avoid confusion or mistakes,
-   * Rush will report an error if this this folder contains any unrecognized files.
-   *
-   * Example: `C:\MyRepo\common\config\rush`
-   */
-  public readonly commonRushConfigFolder: string;
-
-  /**
-   * The folder where temporary files will be stored.  This is always a subfolder called "temp"
-   * under the common folder.
-   * Example: `C:\MyRepo\common\temp`
-   */
-  public readonly commonTempFolder: string;
```

Comment:
Per my other comment, let's bring back `commonRushConfigFolder` and `commonTempFolder` with their old meanings.  We'll change the RFC to say that the "default" subspace gets stored in **common/config/rush** instead of **common/config/subspaces/default**.

Reply:
Brought back `commonRushConfigFolder` and `commonTempFolder`